spec/operators/mergeMapTo-spec.ts

```diff
@@ -264,42 +214,6 @@ describe('Observable.prototype.mergeMapTo', () => {
     expectSubscriptions(e1.subscriptions).toBe(e1subs);
   });
 
-  it('should mergeMapTo many cold Observable, with parameter concurrency=1', () => {
-    const values = {i: 'foo', j: 'bar', k: 'baz', l: 'qux'};
-    const e1 =     hot('-a-------b-------c---|                                        ');
-    const e1subs =     '^                                                            !';
-    const inner =  cold('----i---j---k---l---|                                        ', values);
-    const innersubs = [' ^                   !                                        ',
-                     '                     ^                   !                    ',
-                     '                                         ^                   !'];
-    const expected =   '-----i---j---k---l-------i---j---k---l-------i---j---k---l---|';
-
-    function resultSelector(oV: string, iV: string, oI: number, iI: number) { return iV; }
```

Comment:
ditto

Reply:
exists below "without resultSelector". I refactored to use the pipeable operator anyhow.